app/lib/methods/getUsersPresence.js

```diff
@@ -0,0 +1,65 @@
+import { InteractionManager } from 'react-native';
+import semver from 'semver';
+
+import reduxStore from '../createStore';
+import { setActiveUsers } from '../../actions/activeUsers';
+
+let ids = [];
+
+export default async function getUsersPresence() {
+	const serverVersion = reduxStore.getState().server.version;
+
+	// if server is lower than 1.1.0
+	if (serverVersion && semver.lt(semver.coerce(serverVersion), '1.1.0')) {
+		if (this.activeUsersSubTimeout) {
+			clearTimeout(this.activeUsersSubTimeout);
+			this.activeUsersSubTimeout = false;
+		}
+		this.activeUsersSubTimeout = setTimeout(() => {
+			this.sdk.subscribe('activeUsers');
+		}, 5000);
+	} else {
+		let params = {};
+
+		// if server is greather than or equal 3.0.0
+		if (serverVersion && !semver.lt(semver.coerce(serverVersion), '3.0.0')) {
+			// if not have any id, only subscribe to changes
+			if (!ids.length) {
+				this.sdk.subscribe('stream-notify-logged', 'user-status');
```

Comment:
We're subscribing a lot of times, always that have more one batch of users. It's wrong.